server/py/services/alerts/main.py

```diff
@@ -481,12 +481,19 @@ async def get_alert_activation(
             db_session,
             activation_id,
         )
+        if alert_activation.project != project:
+            raise mlrun.errors.MLRunNotFoundError(
+                f"Alert activation not found. "
+                f"activation_id={activation_id}, "
+                f"project={project}"
+            )
         if name and alert_activation.name != name:
             raise mlrun.errors.MLRunNotFoundError(
                 f"Alert activation not found. "
                 f"activation_id={activation_id}, "
                 f"name={name}"
             )
```

Comment:
reuse the same error message in both errors

Reply:
@TomerShor what if name isn't passed?

Reply:
Oh I missed the difference in project vs name 🤦